[2022-06-08 10:27:58,390] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.install_sqoop manual__2022-06-08T07:26:45.844950+00:00 [queued]>
[2022-06-08 10:27:58,414] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.install_sqoop manual__2022-06-08T07:26:45.844950+00:00 [queued]>
[2022-06-08 10:27:58,415] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 10:27:58,416] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-08 10:27:58,417] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 10:27:58,442] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): install_sqoop> on 2022-06-08 07:26:45.844950+00:00
[2022-06-08 10:27:58,448] {standard_task_runner.py:52} INFO - Started process 30308 to run task
[2022-06-08 10:27:58,453] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'install_sqoop', 'manual__2022-06-08T07:26:45.844950+00:00', '--job-id', '230', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpudxi6_pv', '--error-file', '/tmp/tmpuryvzvn6']
[2022-06-08 10:27:58,456] {standard_task_runner.py:80} INFO - Job 230: Subtask install_sqoop
[2022-06-08 10:27:58,568] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.install_sqoop manual__2022-06-08T07:26:45.844950+00:00 [running]> on host b79bd65fcc57
[2022-06-08 10:27:58,690] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=install_sqoop
AIRFLOW_CTX_EXECUTION_DATE=2022-06-08T07:26:45.844950+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-08T07:26:45.844950+00:00
[2022-06-08 10:27:58,692] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-08 10:27:58,694] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker exec hive-server bash /opt/sqoop/install_sqoop.sh ']
[2022-06-08 10:27:58,708] {subprocess.py:85} INFO - Output:
[2022-06-08 10:27:58,965] {subprocess.py:92} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2022-06-08 10:27:58,967] {subprocess.py:92} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2022-06-08 10:28:03,480] {subprocess.py:92} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 17.1M    0 91929    0     0  71381      0  0:04:11  0:00:01  0:04:10 71373  8 17.1M    8 1507k    0     0   665k      0  0:00:26  0:00:02  0:00:24  665k 42 17.1M   42 7387k    0     0  2263k      0  0:00:07  0:00:03  0:00:04 2263k 88 17.1M   88 15.1M    0     0  3638k      0  0:00:04  0:00:04 --:--:-- 3637k100 17.1M  100 17.1M    0     0  3880k      0  0:00:04  0:00:04 --:--:-- 4123k
[2022-06-08 10:28:22,769] {subprocess.py:92} INFO - mv: inter-device move failed: '/opt/sqoop/sqoop-1.4.7.bin__hadoop-2.6.0' to '/usr/lib/sqoop/sqoop-1.4.7.bin__hadoop-2.6.0'; unable to remove target: Directory not empty
[2022-06-08 10:28:25,107] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-08 10:28:25,156] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=install_sqoop, execution_date=20220608T072645, start_date=20220608T102758, end_date=20220608T102825
[2022-06-08 10:28:25,212] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-08 10:28:25,282] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
